markdownlint-cli2-config-schema.json

```diff
@@ -0,0 +1,83 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"title": "markdownlint-cli2 configuration schema",
+	"type": "object",
+	"properties": {
+		"config": {
+			"$ref": "./node_modules/markdownlint/schema/markdownlint-config-schema.json"
+		},
+		"customRules": {
+			"description": "Module names or paths of custom rules to load and use when linting",
+			"type": "array",
+			"items": {
+				"description": "Module name or path of a custom rule",
+				"type": "string"
+			}
+		},
+		"fix": {
+			"description": "Whether to enable fixing of linting errors reported by rules that emit fix information",
+			"type": "boolean",
+			"default": false
+		},
+		"frontMatter": {
+			"description": "Regular expression used to match and ignore any front matter at the beginning of a document",
+			"type": "string"
+		},
+		"ignores": {
+			"description": "Glob expressions to ignore when linting",
+			"type": "array",
+			"items": {
+				"description": "Glob expression to ignore",
+				"type": "string",
+				"minLength": 1
```

Comment:
Is this true? I think a 0-length array would be handled correctly?

Reply:
(Same comment for similar lines below.)

Reply:
This applies to the string item inside of the array. I looked up how to verify that a glob expression is valid, the only requirement that I saw in `micromatch` was that the glob input string [must have a length greater than zero](https://github.com/micromatch/picomatch/blob/master/lib/picomatch.js#L47). There is also a [`is-valid-glob`](https://github.com/micromatch/is-valid-glob) library from the `micromatch` developers that also rejects glob expressions with zero length.

Reply:
Cool, I thought it was for array length.